site-builder/src/publish.rs

```diff
@@ -57,6 +58,9 @@ pub struct PublishOptions {
     /// See the `list-directory` command. Warning: Rewrites all `index.html` files.
     #[clap(long, action)]
     pub list_directory: bool,
+    /// The maximum number of concurrent calls to the Walrus CLI for the computation of blob IDs.
+    #[clap(long)]
+    max_concurrent: Option<NonZeroUsize>,
```

Comment:
Should we add a default? Whenever someone needs to set up the `max_concurrent`, most likely they will have run into an error.